runtime/isp_kernel.py

```diff
@@ -88,18 +88,24 @@ def runPolicyTool(policy, policies_dir, entities_dir, output_dir, debug):
     
     return True
 
-def buildPolicyKernel(policy, policies_dir, entities_dir, output_dir):
+def buildPolicyKernel(policy, policies_dir, entities_dir, output_dir, arch):
     engine_output_dir = os.path.join(output_dir, "engine")
 
     num_cores = str(multiprocessing.cpu_count())
     with open(os.path.join(output_dir, "build.log"), "w+") as buildlog:
         subprocess.Popen(["make", "-j"+num_cores, "-f", "Makefile.isp"], stdout=buildlog, stderr=subprocess.STDOUT, cwd=engine_output_dir).wait()
 
-    validator_path = os.path.join(engine_output_dir, "build", "librv32-renode-validator.so")
+    val_name = "librv32-renode-validator.so"
+    if arch == "rv64":
+        val_name = "librv64-renode-validator.so"
+
+    validator_path = os.path.join(engine_output_dir, "build", val_name)
     if not os.path.isfile(validator_path):
         return False
 
-    shutil.move(validator_path, output_dir)
+    validator_output_path = output_dir + "/librv32-renode-validator.so"
```

Comment:
Why not change QEMU to link to either `rv32-renode-validator` or `rv64-renode-validator`?

Reply:
It was quicker to just change the name here, but now that it's working it should be simply to make the QEMU change.